xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -4738,13 +4758,10 @@ void CVideoPlayer::UpdatePlayState(double timeout)
     m_processInfo->SetStateRealtime(realtime);
   }
 
-  // FIXME - This block of code only makes sense if the item has EDL *cuts*
-  // not any of the other edit types (mute, commbreak, etc). We loop over
-  // the EDL edit list twice unnecessarily.
-  if (m_Edl.HasEdits())
+  if (m_Edl.HasCuts())
   {
-    state.time = (double) m_Edl.RemoveCutTime(llrint(state.time));
-    state.timeMax = (double) m_Edl.RemoveCutTime(llrint(state.timeMax));
+    state.time = static_cast<double>(m_Edl.GetTimeWithoutCuts(std::lround(state.time)));
+    state.timeMax = std::lround(state.timeMax - static_cast<double>(m_Edl.GetTotalCutTime()));
```

Comment:
This seems wrong, `timeMax` is a `double`, did you mean to use `std::round` instead (or nothing at all perhaps)?

Reply:
Round to the nearest integer according to the original implementation (llrint). lround was your recommendation on the last review

Reply:
Yea sure, it works for the previous line but on this line do you even need to round at all?